org/istio.yaml

```diff
@@ -803,6 +804,7 @@ orgs:
                 members:
                 - costinm
                 - hzxuzhonghu
+                - nrjpoddar
```

Comment:
can you remove your name from this section?. This is for the low level implementation details in EDS code and the core networking code in v1alpha3 folder for listeners/filter chains/etc - the current members (subset of larger networking maintainers) are people who have contributed extensively to code here and are familiar with with the control flow/structure, tradeoffs etc.

Reply:
I see. Will do.

Reply:
Done. PTAL.